NewDawn/NewDawn/QuestionAnswerViewController/QuestionViewController.swift

```diff
@@ -55,6 +57,47 @@ class QuestionViewController: UIViewController {
         return [QuestionAnswer]()
     }
     
+    func removeQuestionAnswerFromLocalStore(q_id: Int) -> Void {
+        // Remove the answered question from local storage
+        if var existed_question_answers: Array<QuestionAnswer> = localReadKeyValueStruct(key: QUESTION_ANSWERS) {
```

Comment:
I saw you declared QUESTION_ANSWERS in answer question view controller. Should we later make constants file to store all the public variable?

Reply:
Yes we should, at least for constants serving as key in local storage. This can be a good follow-up issue.